src/Microsoft.Identity.Web.Certificate/DefaultCertificateLoader.cs

```diff
@@ -74,6 +75,29 @@ public static string? UserAssignedManagedIdentityClientId
 
             return certDescription?.Certificate;
         }
+        
+        /// <summary>
+        /// Load the first certificate from the certificate description list.
+        /// </summary>
+        /// <param name="certificateDescriptions">Description of the certificates.</param>
+        /// <returns>First certificate in the certificate description list.</returns>
+        public static async Task<X509Certificate2?> LoadFirstCertificateAsync(IEnumerable<CertificateDescription> certificateDescriptions)
```

Comment:
Not for this PR, but long-term we should add an overload that accepts a CancellationToken, which should be propagated down - LoadCredentialsIfNeededAsync uses a semaphore which accepts a cancellation token.